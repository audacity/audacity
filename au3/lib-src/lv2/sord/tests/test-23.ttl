# Test long literal
@prefix :  <http://example.org/ex#> .
:a :b """John said: "Hello World!\"""" .
