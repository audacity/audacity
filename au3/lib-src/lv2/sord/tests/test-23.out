<http://example.org/ex#a> <http://example.org/ex#b> "John said: \"Hello World!\"" .
